shared/src/commonMain/kotlin/dev/johnoreilly/confetti/decompose/VenueComponent.kt

```diff
@@ -54,15 +52,44 @@ class DefaultVenueComponent(
             if (initial) {
                 repository.conferenceVenue(conference, FetchPolicy.CacheFirst).data?.let {
                     it.venues.firstOrNull()?.let { venue ->
-                        channel.send(VenueComponent.Success(venue))
+                        val data = Venue(
+                            id = venue.id,
+                            name = venue.name,
+                            address = venue.address,
+                            description = venue.description,
+                            latitude = venue.latitude,
+                            longitude = venue.longitude,
+                            imageUrl = venue.imageUrl,
+                            floorPlanUrl = venue.floorPlanUrl,
+                            mapLink = buildMapLink(venue.name, venue.address)
+                        )
+                        channel.send(VenueComponent.Success(data))
                     }
                 }
             }
-            repository.conferenceVenue(conference, FetchPolicy.CacheFirst).data?.let {
```

Comment:
I removed this code because it conflicted with the if statement on L.52. Even if the condition is false, we still query the repository.